### ISSUE TITLE
Different learning rate for different classes of parameters

### ISSUE THREAD

**Issue by gravity1989** [fchollet/keras#2857]
In the usual case of convolution networks, all the weights have a similar learning rate.

Suppose, I want to divide my weights from a layer into 2 classes: W1 and W2 how can i set different learning rates for the 2 classes of weights?
